README.md

```diff
@@ -32,3 +32,10 @@ It will read the contents of `config.yml` and begin its loop. You could also run
 ### Configuring
 
 In this repo, you can explore the `sample-config.yml` file for an example, but the general structure is as follows. It should be noted that environment variable interpolation happens on load of the YAML file. Also, when alerts are executed, they will be passed through Python's format function with arguments for some attributes of the Monitor. Currently this is limited to `{monitor_name}`.
+
```

Comment:
I see. The issue with applying is that it looks like you pulled my changes and applied your changes as an amend.

You could try again by taking a new branch for your feature and then doing a `git reset --soft upstream/master`. Assuming my repo is your `upstream` remote, this should reset the references to my master, but leave all your current code in tact. Then you can re-commit it to this new branch and send a PR from that branch to my master.

Or, I could just cherry pick the parts of this that have the feature in it.